anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/util/DetectorResource.java

```diff
@@ -30,4 +30,5 @@
 public class DetectorResource {
     private String uuid;
     private ModelTypeResource type;
+    private Boolean enabled;
```

Comment:
Can you clarify what this is for, and how it gets set?

Reply:
Detectors are get disabled through and endpoint exposed by model-service, default is true, it's set to false when someones deletes detector from UI